nixos/modules/services/mail/postfix.nix

```diff
@@ -604,6 +516,152 @@ in
               ])
             );
           options = {
+            mydestination = mkOption {
+              type =
+                with types;
+                nullOr (oneOf [
+                  str
+                  (listOf str)
+                ]);
+              default = [
+                "$myhostname"
+                "localhost.$mydomain"
+                "localhost"
+              ];
```

Comment:
Do we definitely want to be hardcoding these defaults? What if postfix changes them? Wouldn't null be a better default?

Reply:
If Postfix changes the default we'll update ours.